src/view/screens/AppPasswords.tsx

```diff
@@ -260,6 +258,23 @@ function AppPassword({
         </Text>
       </View>
       <FontAwesomeIcon icon={['far', 'trash-can']} style={styles.trashIcon} />
+      <Prompt.Outer control={control}>
+        <Prompt.Title>
+          <Trans>Delete app password?</Trans>
+        </Prompt.Title>
+        <Prompt.Description>
+          <Trans>Are you sure you want to delete the app password</Trans> "
+          {name}"?
```

Comment:
Sorry to jump in here, could I suggest that all of this string is marked for localisation? As I was reminded recently while updating the French translation, the sentence structure can be different in other languages, there can be differences such as _guillemets_ being used instead of quotation marks and spaces can be needed before certain punctuation marks like `?`

```suggestion
          <Trans>Are you sure you want to delete the app password "
          {name}"?</Trans>
```

Reply:
Heh yea, I forget lingui handles that. Thanks!